gulpfile.js

```diff
@@ -237,6 +237,9 @@ gulp.task('dev', ['default'], function() {
   gulp.watch('./app/docs/src/**/*.js', ['rebuild-js-docs']);
   gulp.watch('./app/**/*.html', ['pre-render', 'pre-render-docs']);
   gulp.watch('./app/static/**/*', ['statics', 'statics-docs']);
+  gulp.watch('./type-definitions/*', ['typedefs', 'rebuild-js-docs'], function () {
```

Comment:
rebuild-js-docs already calls `browserSync.reload()` so I don't think you need the callback. However you do probably want to ensure `typedefs` are rebuild first, use `sequence` to have them done sequentially rather than in parallel